CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+- Changed recommended usages to no longer wrap the whole invocation in Sensitive as
+  it is generally not needed.
```

Comment:
this is really confusing, and I don't see the connection to the docs
what is "the whole invocation"?
should this just be "Clarified documentation on usage of `Sensitive` data types"?

Reply:
Maybe it needs a rewording but the main usage guidance changed from:
```puppet
$dbpass = Sensitive(Deferred(conjur::secret, ['production/postgres/password']))
```
to
```puppet
$dbpass = Deferred(conjur::secret, ['production/postgres/password'])
```

Reply:
right. so if we're talking about the 3.0.0 release docs, this maybe doesn't even need a changelog entry? the change from v2.x is that we now require `deferred`

Reply:
This would get removed in 3.0.0 changelog PR since it would be collapsed to only the relevant changes since v2 but it makes sense for this particular version of code on master.

Reply:
ok. let's not forget to drop it, though.

Reply:
👍 https://app.zenhub.com/workspaces/community-and-integrations-team-5e28ab8f700a191286d5abe0/issues/cyberark/conjur-puppet/219#issuecomment-693659196